Merge Jacob's changes

forus opened the pull request:
Minor changes to events upload

jacobrousseau committed:
Ongoing work on the autonomous upload: finished the automatic scheduling

jacobrousseau committed:
Addtion of the stub of the FileCopyService

jacobrousseau committed:
Added the first version of the FileCopyService which manages the file traffic of the
autonomous upload

jacobrousseau committed:
Ongoing work. Removed the workdirectory and addtion of a timestamp to the
file name when copying it to the 'completed' directory

jacobrousseau committed:
Ongoing work on the validation of the CRF-version present in the study compared against the
version(s) in the data

jacobrousseau committed:
Check-in for validation of CRF-version in data compared to the existing CRF-version(s) in
OpenClinica. Changed the list of ItemDefinitions from a java.util.List to a java.util.Set and added a equals / hash to
eliminiate duplicate items.

jacobrousseau committed:
Fix of the broken build.

Commit:
Make get study metadata example xml more readable.

PiotrZakrzewski commented on src/main/java/nl/thehyve/ocdu/OcduApplication.java:
```diff
@@ -8,12 +8,18 @@
 import java.util.Date;
 
 @SpringBootApplication
+@EnableAutoConfiguration
```
I am not sure if this is necessary - AFAIK @SpringBootApplication annotation already contains it

PiotrZakrzewski replied:
Okay I checked: http://docs.spring.io/spring-boot/docs/current/reference/html/using-boot-using-springbootapplication-annotation.html
@EnableAutoConfiguration can be deleted

forus commented:
@jacobrousseau Hi! I've created this PR with your changes consolidated with the master version.

Commit:
studyWsdl.wsdl and it's generated code.

Commit:
eventWsdl.wsdl and it's generated code.

Commit:
studyEventDefinitionWsdl.wsdl and it's generated code.

Commit:
dataWsdl.wsdl and it's generated code.

Commit:
Remove unnecessary annotation.

Commit:
Add parsing of study oid and name.

Commit:
Add event data validations.

Commit:
Add new field to the rest of wsdl-s files.

All wsdl has common bean declarations. One of the wsdl files had the recent change in the common declarations and other were outdated.
The same java source files get generated for each wsdl file! They files replace each other depending on the order of generation wsdl files.

Commit:
Add missing import statements.
